doc/integrations/spark.ipynb

```diff
@@ -0,0 +1,872 @@
+      "text/plain": [
+       "<Figure size 640x480 with 1 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "result = %sql SELECT trip_distance FROM taxi\n",
+    "\n",
+    "import matplotlib.pyplot as plt\n",
+    "\n",
+    "data = result.toPandas()\n",
+    "\n",
+    "plt.boxplot(data[\"trip_distance\"])\n",
+    "plt.xlabel(\"Trip Distance\")\n",
+    "plt.ylabel(\"Value\")\n",
+    "plt.title(\"Boxplot of Trip Distance\")\n",
+    "plt.show()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Persist - Not Supported"
```

Comment:
Please move this to : https://jupysql--968.org.readthedocs.build/en/968/integrations/compatibility.html